internal/providers/pluginfw/tfschema/single_nested_block.go

```diff
@@ -0,0 +1,109 @@
+package tfschema
+
+import (
+	"fmt"
+
+	"github.com/databricks/terraform-provider-databricks/common"
+	dataschema "github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+)
+
+// SingleNestedBlockBuilder represents a list of complex (non-primitive) types.
```

Comment:
Not a list, right? Just a single one?

Is this still used today? Should we even include it if not?

Reply:
Yes just a single one and right not used for now